streampipes-client-python/streampipes/endpoint/api/version.py

```diff
@@ -109,11 +109,20 @@ def get(self, identifier: str, **kwargs) -> Resource:
         identifier: str
             Not supported by this endpoint, is set to an empty string.
 
+        Raises
+        ------
+        NotImplementedError
+            Non-empty 'identifier' is not supported by this endpoint. Please set 'identifier' to an empty string.
+
         Returns
         -------
         versions: Version
             The specified resource as an instance of the corresponding model class([Version][streampipes.model.resource.Version]).  # noqa: 501
         """
+        if identifier != "":
+            raise NotImplementedError(
```

Comment:
I suggest to use `ValueError`
This is not the intended use for `NotImplementedError` (https://docs.python.org/3/library/exceptions.html#NotImplementedError).

Reply:
Thank you for your feedback! I understand your point and will modify the code as per your suggestion.